phoenix-core/src/main/java/org/apache/phoenix/schema/PTable.java

```diff
@@ -341,4 +345,13 @@ public static LinkType fromSerializedValue(byte serializedValue) {
      */
     int getRowTimestampColPos();
     long getUpdateCacheFrequency();
+
+    boolean isNamespaceMapped();
+
+    /**
+     * For a view, return the name of table in Phoenix that physically stores data.
+     * 
+     * @return the name of the Phoenix table storing the data.
+     */
+    PName getPhoenixPhysicalName();
```

Comment:
getPhoenixPhysicalName():- This is needed only in case of views when we want phoenix representation(X.Y) of data table.

getPhysicalName():- this is to get physical/hbase representation(X:Y) of any data table/index.